src/Esri.ArcGISRuntime.Toolkit/UWP/UI/Controls/SymbolDisplay/SymbolDisplay.Theme.xaml

```diff
@@ -7,7 +7,7 @@
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="controls:SymbolDisplay">
-                    <Image x:Name="image" />
+                    <Image x:Name="image" Stretch="Fill" MaxWidth="40" MaxHeight="40"/>
```

Comment:
MaxWidth/Height should be on the SymbolDisplay control, not it's internal items.
Ie
```
<Setter Property="MaxHeight" Value="40" />
```
However I don't think we should make that default. We should display the symbol the user gave us, and they are free to set `<esri:SymbolDisplay MaxHeight="40" />`